listener.py

```diff
@@ -98,53 +125,47 @@ def compileTest(codeText: str, version: str):
             fc.write(loadTemplate(codeText))
         else:
             fc.write(loadTemplate(codeText, False))
+    command = [
+        "bash",
+        "-c",
+        "cp code/* .; DreamMaker test.dme; DreamDaemon test.dmb -close -verbose -ultrasafe | cat",
+    ]
```

Comment:
This is pretty messy compared to just having the script copied into the dockerfile.

Reply:
Ah right, I knew there was a reason I didn't want to PR my crap code. Will try to figure out how to do this all properly.